service/src/main/resources/logback.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is very similar to the default spring configuration, with an addition to the console log pattern to include values from the [MDC](https://logback.qos.ch/manual/mdc.html) (requestId, user). Technically we could do without this as user will still get logged to cloud logs, but for consistency and local testing I think it's beneficial to include this in our console logs, in addition to any other request information we may want to log in the future.

Reply:
Just to clarify, does this change the logging pattern for only local environment or both local and live environments?

Reply:
This file is only applicable to local "human-readable" logs. The logic for including this information in the live logs is handled by terra-common-lib and is included the same way as requestId. You can also see this locally by turning off the human-readable-logging profile